### ISSUE TITLE
IE10 disables resize slider after image is loaded

### ISSUE THREAD

**Issue by SparserLogic** [scottcheng/cropit#37]
You can reproduce in the basic demo. As soon as an image is selected, the slider is disabled.

**Comment by scottcheng** [comment]
@SparserLogic Sorry but I don't have access to IEs right now. Any chance you could check if there's any error in the console?

**Comment by piotrze** [comment]
@scottcheng It is caused by lack of css styles for range input for IE.

**Comment by BugHunter2k** [comment]
@scottcheng For testing in IE you can grab a virtual pc/virtualbox image from Microsoft:
https://www.modern.ie/en-us/virtualization-tools#downloads

**Comment by scottcheng** [comment]
Thanks for the input @piotrze @BugHunter2k.

@SparserLogic I tried it on Browserstack -- the slider functions on IE10+, but I agree the style makes it quite unusable. A few months ago I spent some time tweaking `range` input styles in IE, and it was really frustrating. I'd say if you want to have consistent styling across IEs and other browsers, feel free to use a customized slider instead of a native one. I'll make it clear on the website.
